Automattic/jetpack
# Boost: Add toast notification when toggling cache logging

Part of https://github.com/Automattic/jetpack/issues/35768

## Proposed changes:

* Add a toast notification when enabling/disabling cache logging

![image](https://github.com/Automattic/jetpack/assets/11799079/17fef66d-2cb3-40c1-a53a-3a62d8bc0bc1)

![image](https://github.com/Automattic/jetpack/assets/11799079/fb8b20f4-2ba5-4d94-80af-e8dfaa528455)

* Add a notification manager to fix overlapping notifications.

https://github.com/Automattic/jetpack/assets/11799079/728d752f-6c6f-44a9-87d8-34486a0a490c

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Toggle logging on and off - you should see the notification show up;
* There should be no overlapping notifications.
